test/test.sh

```diff
@@ -49,9 +49,9 @@ if test -n "$CAN_SUDO"; then
 		sudo pfctl -a pgbouncer -F all -q 2>&1 | grep -q "pfctl:" && {
 			cat <<-EOF
 			Please enable PF and add the following rule to /etc/pf.conf
-			
```

Comment:
remove or make separate commit

Reply:
I should remove my branch and recreate it for create a separate commit? I'm not a git expert.